tests/sentry/utils/test_integrationdocs.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from sentry.utils.integrationdocs import dump_doc, load_doc
+
+
+@pytest.mark.parametrize(
+    "path",
+    [
+        ("/"),
+        ("/.."),
+        ("//...."),
+        ("/%5c.."),
+        ("../"),
+        ("../../"),
+        ("../../../etc/passwd"),
+    ],
+)
+def test_path_traversal_attempt_on_load_doc_raises_exception(path):
+    with pytest.raises(Exception):
+        load_doc(path)
+
+
+@pytest.mark.parametrize(
+    "path",
+    [
+        ("/"),
+        ("/.."),
+        ("//...."),
+        ("/%5c.."),
+        ("../"),
+        ("../../"),
+        ("../../../etc/passwd"),
+    ],
+)
+def test_path_traversal_attempt_on_dump_doc_raises_exception(path):
+    data = {"foo": "bar", "baz": 1234}
+
+    with pytest.raises(Exception):
```

Comment:
Done! I had to introduce a custom exception into `sentry.utils.integrationdocs` instead of using `django.core.exceptions` ([see the note at the top of the file](https://github.com/getsentry/sentry/blob/master/src/sentry/utils/integrationdocs.py#L1-L5)), but either way, I like the more targeted approach. Thank you for the feedback!